hopp/simulation/technologies/steel/enthalpy_functions.py

```diff
@@ -0,0 +1,194 @@
+## Functions to calculate enthalpy of different streams 
+
+### Hydrogen enthalpy coefficients
+# The value of enthalpy returned is the specific enthalpy in kj/g
+'''
+Author: Charlie Kiefer
+Date: 8/14/23
+
+These functions calculate enthalpies of elements relating to green steel
+
+Sources:
+Chase, M.W., Jr. 1998. "NIST-JANAF Themochemical Tables, Fourth Edition." J. Phys. Chem. Ref. Data, Monograph 9 1-1951. doi:https://doi.org/10.18434/T4D303.
+    Thermochemistry tab of each element
+
+Inputs:
+    T: Temperature of the element (K)
+
+Outputs:
+    H_t: Enthalpy of element at given temp (kj/g)
+'''
+
+def h2_enthalpy_1(T):
+    mol_weight_H2=2.01588 #in grams# T1 and T2 should be in the range of 298-1000 K <br>
+    t=T/1000 
+    A=33.066718 
+    B=-11.363417
+    C=11.432816 
+    D=-2.772874 
+    E=-0.158558 
+    F=-9.980797 
+    G=172.707974 
+    H=0  
+    H_t=A*t +(B*t*t)/2 +(C*t*t*t)/3 + (D*t*t*t*t)/4-(E/t)+F-H 
+    return H_t 
+def h2_enthalpy_2(T):
+    mol_weight_H2=2.01588#T in range 1000-2500 K 
+    t=T/1000 
+    A=18.563083 
+    B=12.257357 
+    C=-2.859786  
+    D=0.268238 
+    E=1.977990 
+    F=-1.147438 
+    G=156.288133 
+    H=0 
+    H_t=(A*t +(B*t**2)/2 +(C*t**3)/3 + (D*t**4)/4-(E/t)+F-H)/mol_weight_H2 
+    return H_t 
+### Water enthalpy coefficients
+
+def h2o_enthalpy(T):
+    mol_weight_H2O=18.0153 
+    t=T/1000 
+    A=30.09200  
+    B=6.832514 
+    C=6.793435 
+    D=-2.534480 
+    E=0.082139 
+    F=-250.8810 
+    G=223.3967 
+    H=-241.8264 
+    H_t=(A*t +(B*t*t)/2 +(C*t*t*t)/3 + (D*t*t*t*t)/4-(E/t)+F-H)/mol_weight_H2O 
+    return H_t 
+
+def fe_enthalpy_1(T):#298-1809 K
```

Comment:
same comment on using a single function as I gave in the h2_enthalpy_x functions

Reply:
Merged into one function with if statements